src/adafruit_blinka/microcontroller/amlogic/s905x/pin.py

```diff
@@ -53,17 +53,17 @@
 UART2_RX = GPIO292
 UART2_TX = GPIO291
 
-SPI1_SCLK = GPIO223
-SPI1_MISO = GPIO288
-SPI1_MOSI = GPIO287
-SPI1_CS = GPIO289
+SPI0_SCLK = GPIO290
+SPI0_MISO = GPIO288
+SPI0_MOSI = GPIO287
+SPI0_CS = GPIO289
 
 i2cPorts = (
     (0, I2C0_SCK, I2C0_SDA),
     (1, I2C1_SCK, I2C1_SDA),
 )
 
-spiPorts = (0, SPI1_SCLK, SPI1_MOSI, SPI1_MISO)
+spiPorts = [(0, SPI0_SCLK, SPI0_MOSI, SPI0_MISO)]
```

Comment:
```suggestion
spiPorts = (
    (0, SPI0_SCLK, SPI0_MOSI, SPI0_MISO),
    (1, SPI1_SCLK, SPI1_MOSI, SPI1_MISO)
)
```

Reply:
https://learn.adafruit.com/adding-a-single-board-computer-to-blinka/getting-spi-working

From the link above 

At this time, only Spidev 0 is supported. ??

Reply:
Yeah, meaning spidev0.0 and spidev0.1, but not spidev1.0 or spidev1.1. I'll try and make that a bit clearer. :)